styles.css

```diff
@@ -0,0 +1,3 @@
+.h1{
+    color: blue;
+}
```

Comment:
- [ ] Your design 👇 

![hello](https://user-images.githubusercontent.com/78251927/205367436-bdcda8b0-eea4-4a0e-a0a8-297c84ff2877.PNG)

**[OPTIONAL]** As this requirement was not mentioned by the previous reviewer, it is marked optional, but recommended. please kindly consider making the text color effective. I see that you used `.h1` in your CSS file instead of using just `h1` selector. as your `h1` tag does not have `class=h1` in it. 😃
```
<h1> Hello Microverse </h1>
```